test/ng/directive/ngRepeatSpec.js

```diff
@@ -1566,6 +1572,107 @@ describe('ngRepeat animations', function() {
       item = $animate.queue.shift();
       expect(item.event).toBe('move');
       expect(item.element.text()).toBe('3');
+
+      item = $animate.queue.shift();
+      expect(item.event).toBe('move');
+      expect(item.element.text()).toBe('1');
+    })
+  );
+
+  it('should fire off the move animation for filtered items',
```

Comment:
How about this description?
`should fire off the move animation for items that change position when other items are filtered out`